lexicons/tools/ozone/moderation/defs.json

```diff
@@ -131,6 +131,21 @@
           "format": "datetime",
           "description": "Timestamp referencing the first moderation status impacting event was emitted on the subject"
         },
+        "recordUpdatedAt": {
+          "type": "string",
+          "format": "datetime",
+          "description": "Last update timestamp of the record the subject is associated with"
+        },
+        "recordDeletedAt": {
+          "type": "string",
+          "format": "datetime",
+          "description": "Timestamp referencing when the record the subject is associated with was deleted"
+        },
+        "recordStatus": {
+          "type": "string",
+          "description": "Status of the record the subject is associated with. Statuses are different when the subject references an account vs. a record",
+          "knownValues": ["takendown", "suspended", "deleted", "deactivated"]
+        },
```

Comment:
This is quite a complex field, I think there are a few potential issues with it:

1. It's called  "record status" but it may contain the hosting status of an account.
2. The hosting statuses of records and accounts are mixed together, which could make it seem like a record can be marked as takendown here, which isn't the case.
3. Potential confusion between the ozone status and the hosting status: the terms "takendown" and "suspended" are also terms used by ozone overlapping with the `takendown` and `suspendedUntil` fields.

Here's one idea: we could combine the three new fields into something more like this (using typescript just to illustrate the concept):
```ts
type SubjectStatusView = {
  // ...
  hosting?: AccountHosting | RecordHosting
}

type AccountHosting = {
  $type: 'tools.ozone.moderation.defs#accountHosting',
  status: 'active' | 'takendown' | 'suspended' | 'deleted' | 'deactivated'
  createdAt: Date
  updatedAt: Date
  deletedAt?: Date
  suspendedAt?: Date
  deactivatedAt?: Date
  takendownAt?: Date
  reactivatedAt?: Date
}

type RecordHosting = {
  $type: 'tools.ozone.moderation.defs#recordHosting'
  status: 'active' | 'deleted'
  createdAt: Date
  updatedAt: Date
  deletedAt?: Date
}
```

This also gives us a way to make certain fields required together if we want, e.g. if `hosting` is present then `hosting.createdAt` can always be present too.

Reply:
this is nice!